### DISCUSSION TITLE
Under which licence is the tiny IXI Dataset distributed and can it be used under modification?

### DISCUSSION THREAD

**Discussion by sssilvar** [TorchIO-project/torchio#864] [Q&A] [answered]
We are trying to adapt and use the Tiny IXI dataset you provided for [segmentation tasks](https://colab.research.google.com/github/fepegar/torchio-notebooks/blob/main/notebooks/TorchIO_tutorial.ipynb#scrollTo=VqyNi7QuiDWv) and we wonder if this dataset is open and reusable under CC.

**Comment by fepegar** [comment, marked as answer]
Hi, @sssilvar,

Hm. IANAL, but I think you can use it freely. If you publish something with it, the authors of IXI say

> This data is made available under the Creative Commons [CC BY-SA 3.0 license](https://creativecommons.org/licenses/by-sa/3.0/legalcode). If you use the IXI data please acknowledge the source of the IXI data, e.g. this website.

If you use `IXITiny`, TorchIO should be appropriately [acknowledged](https://torchio.readthedocs.io/index.html#credits) as well.
